ocs_ci/helpers/helpers.py

```diff
@@ -838,6 +838,47 @@ def validate_cephfilesystem(fs_name):
     return True if (ceph_validate and ocp_validate) else False
 
 
+def create_ocs_object_from_kind_and_name(
+    kind, resource_name, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+):
+    """
+    Create OCS object from kind and name
+
+    Args:
+        kind (str): resource kind like CephBlockPool, pvc.
+        resource_name (str): name of the resource.
+        namespace (str) the namespace of the resource.
+
+    Returns:
+        ocs_ci.ocs.resources.ocs.OCS (obj): returns OCS object from kind and name.
+
+    """
+    ocp_block_object = OCP(
```

Comment:
A few comments:
1) ocp_block_object falsely indicates that this function creates a specific kind of object 
2) why can't we use an existing function, such as ocs_ci.helpers.helpers.create_resource or one that creates an object of a specific kind?

Reply:
1. Didn't understand quiet but in the end I'm returning OCS object as the function describes.
2. In ocs_ci.helpers.helpers.create_resource you have to pass a dictionary and since I've getting data from UI I prefer to have it this way with 3 arguments and not create a dict.

Reply:
1) what does `ocp_block_object` var name mean?
2) can't we extend ocs_ci.helpers.helpers.create_resource to allow it to get  a dict? I just think that maintaining another function that does pretty much the same is a redundancy

Reply:
1. Got it. Changing the name to ocp_object :)
2. ocs_ci.helpers.helpers.create_resource is different than create_ocs_object_from_kind_and_name:
a. ocs_ci.helpers.helpers.create_resource really creates a resource on the cluster while my function is creating an 
  object(python) and not really creating the resource on the cluster. It is for an already existing resource that was created from UI and is translating the name, kind and namespace into an python object. In cli tests you already have an object if you created a resource. Here the resource is created in UI but I we need to have a representation of it in the code so I'm creating an OBJECT.
 It is complex to merge the two function and will always have to do two steps in UI test. First create the dict with OCP and then decide if the user wants to just create a python object or really create resource on the cluster. I don't think the 2 function are the same.

Reply:
Ok, I understand. Please just check maybe we already have a function that does the same thing

Reply:
![image](https://user-images.githubusercontent.com/11279672/133102232-1332e76e-4c81-4fdd-aafc-021926e6ba9d.png)
I didn't find one with the relevant words